### PR TITLE
fix: define get_distance_measure by default

### PR DESCRIPTION
**Pull request by barendgehrels**
This fixes circleci
https://app.circleci.com/pipelines/github/boostorg/geometry/257/workflows/00234789-5f80-4b23-9075-4c359ec83d11/jobs/15405

### CODE REVIEW

**Comment by barendgehrels** [comment]
Merged to get ci green

**Review by vissarion — APPROVED** [review]

**Comment by vissarion** [comment]
Sorry, I was a bit late, I am OK in any case.

File: include/boost/geometry/algorithms/detail/overlay/get_distance_measure.hpp [hunk 1]
```diff
@@ -61,13 +61,23 @@ namespace detail_dispatch
 
 template <typename CalculationType, typename CsTag>
 struct get_distance_measure
-        : not_implemented<CsTag>
```
Comment (awulkiew) [inline include/boost/geometry/algorithms/detail/overlay/get_distance_measure.hpp:64]: This struct is now going to be instantiated for all geometries defining CSTag other than cartesian_tag, also for cs_undefined_tag regardless of the strategy passed by the user. This is incorrect.

**Comment by awulkiew** [comment]
Are we abandoning the commit message format?

**Comment by barendgehrels** [comment]
> Are we abandoning the commit message format? I'm ok with it. What is this format exactly? Prefixing with `fix:` and `feature:` or something else too?

Sorry, I see this only now. Now, I by accident used a different format from another project. 
Usually I use `[subject] Message` for example `[test] Fixed expectations` or `[buffer] Implemented for geographic` or so.

**Pull request merged**